client/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "client",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
```

Comment:
next lint wont take from custom eslint used above. Use our own eslint run command. Also above config has some issues. Run the eslint command and you will get errorrs

Reply:
> next lint wont take from custom eslint used above. Use our own eslint run command. Also above config has some issues. Run the eslint command and you will get errorrs

##  when i add some unused js line eslint not showing error need help